.github/workflows/release.yaml

```diff
@@ -72,22 +72,11 @@ jobs:
 
       - name: Install dependencies
         run: |
-          cargo install wasm-bindgen-cli
+          cargo install trunk
 
-      - name: Build binary
+      - name: Build Wasm
```

Comment:
Why rename this? Wasm is still a binary, no?

Reply:
Yeah, but trunk does more than just building the binary. It also runs `wasm-bindgen`, moves assets, builds the actual `index.html`, etc. I see it more as a "Build entire Wasm package"

Reply:
I would name this `Prepare package` then.